### ISSUE TITLE
#1033 Deepfake Sites

### ISSUE THREAD

**Issue by imurreflection** [dobbelina/repository.dobbelina#1067]
Hi Dobbelina,

I can see that you have c;losed the above issue #1033 as "Completed" . May I request you to provide custom sites here please:

https://dobbelina.github.io/custom.html

I tried copying the code mentioned in above issue but somehow it doesnt work. Just curious to check how it looks. Checked one such video of Gal Gadot and i was surprised to see the accurateness of DF sites.

**Comment by ErosVece** [comment]
I can explain the `somehow it doesn't work` 🤣 
It also says above the code, that it's gpt generated :)

It's missing stuff like the Playvid function. And a big change there is more wrong with it.

From what I've heard is that the people from Team Cumination don't do custom site requests. 
But any other member is welcome to create and share the custom site himself

**Comment by imurreflection** [comment]
I apologize sir. It was mentioned in the issue that the code was corrected so i thought creating a py file or zip file might work but that's not the case. Sir, you are an expert and i leave this to your jurisdiction. Just wanted to let you know that the site has proper DMCA information so i don't think there could b any legal tussle. Rest i leave it to you sir. Have a nice day

**Comment by ErosVece** [comment]
> I apologize sir. It was mentioned in the issue that the code was corrected so i thought creating a py file or zip file might work but that's not the case. Sir, you are an expert and i leave this to your jurisdiction. Just wanted to let you know that the site has proper DMCA information so i don't think there could b any legal tussle. Rest i leave it to you sir. Have a nice day

You mean that what @dobbelina said?
```
I apologize here is the correct code.....:)
Is there anything else i can help you with today? 😄
```
That is a joke, it's how ChatGPT often responds if you say something is wrong in the code.

**Comment by imurreflection** [comment]
Oh ..i never realized that but then if i may ask why it was closed as "Completed "?  Can you tell me which other site py file can be copied and modified for this site please

**Comment by ErosVece** [comment]
https://github.com/dobbelina/repository.dobbelina/blob/3c61f2b8cfa68751289779184f41b20dda70b89c/plugin.video.cumination/resources/lib/sites/tabootube.py
Tabootube seems to be using the same CMS, I see that the regex for the videos needs a few small tweaks....maybe there is more

**Comment by ErosVece** [comment]
@imurreflection sneak preview from something that might get out.....some day 🤣 
It's only a mockup/wireframe...so this isn't the look

![image](https://github.com/dobbelina/repository.dobbelina/assets/37535332/c9a7a06e-b2c6-408b-a6a6-10db54ca5ca5)

**Comment by imurreflection** [comment]
This is awesome sir. Is there any trial spot available?

**Comment by DJMix-I-Can** [comment]
What's up @dobbelina & @ErosVece & the other contributors. I thought I'd give you guys my quick opinion on what I'm seeing. 
I understand I'm a newbie & still learning. My opinion might not mean much BUT anywho. One of my questions is, why are you guys still tending to those who are using older versions of Kodi? It was stated in the beginning that it wasn't going to be supported anymore. Meanwhile it is... As from the previous mess, that is what happened. Majority of the issues was due to older versions of Kodi. Next thing... Custom sites... I feel that it needs to be thoroughly explained that just changing a domain name from a previous site. Will not make your custom site work. A lot of people are under the assumption that it will, which isn't the case. They're not aware that they need to learn how it works. Constantly asking for a "step-by-step" guide on how to do it, regardless of not understanding that every site is different. The regex codes are different. I've learned that because I took the initiative to learn. A lot of the requests & commentators haven't taken that initiative, except for maybe two users. ErosVece, you have your ideas to try to make it better but in all honesty. A lot of these users don't care about how to correct it themselves. They want you, the other contributors to do it for them. As you stated you're lazy & in all honesty. I feel you're wasting your energy on trying to help those who are not willing to help Cumination. If you feel that you have ideas for it. By all means, do it on the down low instead of advertising it. You'll open up a can of worms & will be asked on the status of it. Ex: What you just posted 4 hours ago.... Personally, I'm going to step away from Cumination.   A lot of the new sites are repetitive material. Repetitive comments from users who don't contribute on anything is getting very annoying. Thank you for allowing me to learn from the devs but I'm done.  Be well....

**Comment by imurreflection** [comment]
Hello DJMix-I-Can,  I know what you are trying to say and what makes you feel disappointed and dis-interested. However, you must understand that every user has its own capability and caliber and may be not everyone is as intelligent as you are. I am trying hard to learn and earlier I was not aware of many things like A) what is .py 2) how to replace files without waiting for updates and others. I am Six Sigma Black Belt certified and that is my area of expertise. This has requirement of some core competency and everyone who comes to me or ask for any help, I try my level best to provide the solution in most effective and friendly manner. No matter how many times; as I love to resolve issues and especially those that challenges them or me.  I have never used statements like 'You'll open up a can of worms '. I am surprised that such a humble person like you can use statements like this. Anyways, this is how we all learn about each other, I can assure you that you will not see any comments or site requests or anything from me ever. However, dont use statements like 'I feel you're wasting your energy on trying to help those who are not willing to help Cumination. ' or 'do it on the down low instead of advertising it.' or 'I'm going to step away from Cumination.' I would request you to use your "EXPERTISE" here and help this community. Be well mate

**Comment by DJMix-I-Can** [comment]
I have dealt with corporate companies that felt they needed someone with a certain "certification" in order to charge more. When it came down to doing hands on work. They were clueless.... Just because you read a book or took an online certification doesn't mean they know anything.....  Anywho... I never stated it was you!!!!! If the shoe fits, then wear it.....  Do me a favor & don't comment on it.

**Comment by ErosVece** [comment]
@DJMix-I-Can I understand what you are saying, and I agree with some stuff.
Yes, everybody should upgrade, but I also understand the reasons when Kodi 18 users won't upgrade.
But the users running 19, should update to 20.

The mess from last time was 100% my fault, it were 2 separate things actually.
One mistake with adding the exception_logger. I tested it with Kodi 19, I mistakenly thought that 19 was Python 2.7, no idea how that happend :)  It worked there, so I thought it worked with Py2.
The other mistake was sending all m3u8 files through Inputstream Adaptive. It doesn't work like that, adaptive streams would be better but the others don't.
It would also broke for example Perverzija, we do something with loading the m3u8 from local drive.
I didn't pay enough attention when I changed it.

About the custom sites, I won't teach anybody how to do it step-by-step, but I'll give pointers how it works or where to find information. The custom sites were always intended for the people who are more experienced. If not, they can try and fail...they'll learn from that.
Some of the users who would ask such things, could contribute in other ways like suggesting sites to add. Some times there is a good one between all the 3 min video junk sites. That is also worth something. I'm not pointing fingers, but who the shoe fits... ;)
Wanting to learn or try, is always a plus. Even if they fail.

That what I'm working on now is far from wasting energy. At first it is for myself, but maybe I'll never used it.
It's a new challenge for me, I learn new stuff (making wireframe/mockup, Customtkinter, and who knows more).
Maybe it will set the bar lower for users who do want to learn.
If you have never seen Python or anything, opening up a sites Python file, could be overwhelming.
With this, it can be a step in the right direction, they can compare what they fill out in the form with where it's place is in the code. 

Me sharing a sneak peak, is because I'm enthusiastic how it's going :)
Maybe it will never get to see the daylight, that I'm stuck and lost interest...we'll see.
It will anyhow be unique, I haven't seen something similar in the Kodi world.
And maybe noone will use it, look at for example that Github Downloader, I have no idea if that is used by 1 person or a few or 100s. I think I never had a question about it :)

So, it's a shame if you'd leave....one of the good ones. Is there a way to contact you, Github is still missing a private message function :)
